File: lab_2/ex2.py
class A:
    def vvod1(self, a, b , c):
        self.a = a
        self.b = b 
        self.c = c

class B:
    def vvod2(self, x, y):
        self.x = x
        self.y = y

class C(A, B):
    def sum(self):
        s = self.a + self.b + self.c + self.x + self.y
        return s
    def mult(self):
        m = self.a * self.b * self.c * self.x * self.y
        return m

tmp = C()
tmp1 = C()

tmp.vvod1(4, 2, 6,)
tmp.vvod2(3, 6)

tmp1.vvod1(1, 1 ,1)
tmp1.vvod2(2, 9)

print("First:")
print("\tSum:", tmp.sum())
print("\tMult:", tmp.mult())

print("Second:")
print("\tSum:", tmp1.sum())
print("\tMult:", tmp1.mult())

File: lab_2/cpp/ex2.cpp
#include <iostream>

using namespace std;

class First {
   public:
    int a, b, c;
    void vvod1() {
        cout << "a = ";
        cin >> a;
        cout << "b = ";
        cin >> b;
        cout << "c = ";
        cin >> c;
    }
};

class Second {
   public:
    float x, y;
    void vvod2() {
        cout << "x = ";
        cin >> x;
        cout << "y = ";
        cin >> y;
    }
};

class Third : public First, public Second {
   public:
    float s, m;
    void sum() {
        s = a + b + c + x + y;
        cout << s << endl;
    };
    void mult() {
        m = a * b * c * x * y;
        cout << m << endl;
    };
};

int main(int argc, char const *argv[]) {
    Third tmp;
    tmp.vvod1();
    tmp.vvod2();
    tmp.sum();
    tmp.mult();

    getchar();
}
